docs/usage/exploring-observations.md

```diff
@@ -1,7 +1,109 @@
-# Manual creations
+# Observations
 
-!!! tip "Under construction"
+When you click on "Observations" in the left-side bar, you access all the "Observations" tabs, visible on the top bar on the left. By default, the user directly access the "Observables" tab, but can navigate to the other tabs as well.
```

Comment:
the user directly accesses

Reply:
I would rather use the verb "land":
By default, the user directly lands on the "Observables" tab